server/models/default_local_provider.go

```diff
@@ -661,14 +661,18 @@ func (l *DefaultLocalProvider) SaveMesheryFilter(_ string, filter *MesheryFilter
 }
 
 // GetMesheryFilters gives the filters stored with the provider
-func (l *DefaultLocalProvider) GetMesheryFilters(_, page, pageSize, search, order string) ([]byte, error) {
+func (l *DefaultLocalProvider) GetMesheryFilters(_, page, pageSize, search, order string, visibility string) ([]byte, error) {
 	if page == "" {
 		page = "0"
 	}
 	if pageSize == "" {
 		pageSize = "10"
 	}
 
+	if visibility == "" {
+		visibility = "private"
```

Comment:
Replace
```go
if visibility == "" {
		visibility = Private
```

Reply:
I see, then I have to use `models vars`.

Also, here is a video with the previous changes considering all the filters were private:

https://github.com/meshery/meshery/assets/64671908/87206349-ccfb-40c2-b262-d92c49a953bd

Reply:
I addressed your request changes.

Can you check it once again?